includes/specials/SpecialDeletedContributions.php

```diff
@@ -137,7 +141,7 @@ function formatRow( $row ) {
 				'id'         => $row->ar_rev_id,
 				'comment'    => $row->ar_comment,
 				'user'       => $row->ar_user,
-				'user_text'  => $row->ar_user_text,
+				'user_text' => User::getUsername( $row->ar_user, $row->ar_user_text ),
 				'timestamp'  => $row->ar_timestamp,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "timestamp" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)